app/scripts/lib/multichain-api/MultichainMiddlewareManager.ts

```diff
@@ -0,0 +1,87 @@
+import { JsonRpcMiddleware } from 'json-rpc-engine';
+import { Scope } from './scope';
+
+// Extend JsonRpcMiddleware to include the destroy method
+export type ExtendedJsonRpcMiddleware = JsonRpcMiddleware<unknown, unknown> & {
+  destroy?: () => void;
+};
```

Comment:
thoughts on adding a short comment about why this is needed OR when this is no longer needed?

Reply:
➕